plugins/foreman_docker/1.x/index.md

```diff
@@ -0,0 +1,251 @@
+
+`Image` is a static snapshot of the containers' configuration. Image is a read-only layer that is never modified, all changes are made in top-most writable layer, and can be saved only by creating a new image. Each image depends on one or more parent images.
+
+`Platform image` an image that has no parent. Platform images define the runtime environment, packages and utilities necessary for containerized applications to run. The platform image is read-only, so any changes are reflected in the copied images stacked on top of it.
+
+`Registry` is a public or private archive that contains images available for download. Some registries allow users to upload images to make them available to others. Foreman allows you to import images from local and external registries. Foreman itself can act as an image registry for hosts, however, hosts cannot push changes back to the registry, other than committed images.
+
+`Tags` are used to differentiate images in a repository, they typically mark the version of the application stored in the image. Repositories are used to group similar images in a container registry. Images only have unique alphanumeric identifiers, so naming in form or repository:tag provides a human-readable way of identifying images.
+
+With Foreman and Katello, you can create an on-premise registry, [import images from various sources and distribute them to containers using content views](http://www.katello.org/docs/2.2/user_guide/docker/docker.html). Foreman supports creating a Docker compute resource, that acts as a server for running containers. This way, you can import an image, start a container based on this image, monitor the container's activity, and commit it's state to a new image layer that can be further propagated.
+
+![Containers list](static/images/plugins/foreman_docker/docker-containers-list.png)
+
+# 2. Installation
+
+### Foreman Installer (recommended)
+You can use the Foreman installer to install this plugin. Use the following command:
+    foreman-installer --enable-foreman-plugin-docker
+
+If you prefer not to use the installer, follow the instructions below for your operating system.
+
+### Red Hat, CentOS, Fedora, Scientific Linux (rpm)
+
+Set up the repo as explained in the link above, then run
+
+    # yum install ruby193-rubygem-foreman_docker
+
+### Debian, Ubuntu (deb)
+
+Set up the repo as explained in the link above, then run
+
+    # apt-get install ruby-foreman-docker
+
+### Bundle (gem)
+
+Add the following to bundler.d/Gemfile.local.rb in your Foreman installation directory (/usr/share/foreman by default)
+
+    $ gem 'foreman_docker'
+
+Then run `bundle install` and `foreman-rake db:migrate` from the same directory
```

Comment:
Nitpick, this sort of muddles a source and package installation a bit, I'd avoid recommending it or prefix this section with a notice to say that it's only for *source* installations.

(foreman-rake's a packaged command, bundle install wouldn't be recommended/wouldn't work on Debian or RPM installations.)